janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/resources/hbase-read.properties

```diff
@@ -0,0 +1,29 @@
+gremlin.graph=org.apache.tinkerpop.gremlin.hadoop.structure.HadoopGraph
+gremlin.hadoop.graphInputFormat=org.janusgraph.hadoop.formats.hbase.HBaseInputFormat
+gremlin.hadoop.graphOutputFormat=org.apache.hadoop.mapreduce.lib.output.NullOutputFormat
+#gremlin.hadoop.graphOutputFormat=org.apache.hadoop.mapreduce.lib.output.SequenceFileOutputFormat
```

Comment:
I just copied this file from cassandra-read.properties which includes the commented out line (probably for no good reason). Let me know if I should remove.